doc/howto/move_instances.md

```diff
@@ -30,8 +32,24 @@ If you need to adapt the configuration for the instance to run on the target ser
 (live-migration)=
 ## Live migration
 
+Live migration means migrating an instance while it is running.
+This method is supported for virtual machines.
+For containers, there is limited support.
+
+(live-migration-vms)=
+### Live migration for virtual machines
+
 Virtual machines can be moved to another server while they are running, thus without any downtime.
 
+To allow for live migration, you must enable support for stateful stop.
```

Comment:
"you must enable support for stateful migration" (which is also required for stateful stop, stateful snapshot and stateful start, so its an umbrella term).